Document the replications settings in app_Settings (#332)
medic/cht-docs: content/en/apps/reference/app-settings/replication.md

---

> title: ".replications"

Reviewer: ```suggestion
title: ".replications [deprecated]"
```

linkTitle: ".replications"
weight: 5
description: >

>   **Replications**: Configure replication of databases to a main meta database.

Reviewer: ```suggestion
  **Replications** [deprecated]: Configure replication of databases to a main meta database.
```

keywords: replications
---

> **Deprecation note: This is only available in versions 3.5.0 to 3.9.0. As of 3.10.0 this no longer configurable and happens nightly for user meta databases to a central meta data database.** 

Reviewer: ```suggestion
{{% alert title="Deprecated" %}} 
The `replications` field is only available in versions 3.5.0 to 3.9.0. As of 3.10.0 this field is ignored, and replication happens nightly for user meta databases to a central meta data database.
{{% /alert %}}
```
